pkg/readers/container_image_reader_test.go

```diff
@@ -0,0 +1,31 @@
+package readers
+
+import (
+	"github.com/safedep/vet/pkg/models"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestContainerImageReaderEnumManifest(t *testing.T) {
```

Comment:
This test will significantly slow down development because it will pull the image. We should move container image reader tests to E2E that runs in CI. Have simple validation tests here that are fast.

In future, we need to see if we can mock the results to test other cases.